package.json

```diff
@@ -49,6 +49,7 @@
     "request": "^2.81.0",
     "rimraf": "^2.6.2",
     "rotating-file-stream": "^1.3.2",
+    "sha256": "^0.2.0",
```

Comment:
This is unpopular and unsupported package. Lets use `hash.js` or the built-in `crypto` package